pyiron_base/jobs/job/runfunction.py

```diff
@@ -1,14 +1,14 @@
 # coding: utf-8
 # Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
 # Distributed under the terms of "New BSD License", see the LICENSE file.
-from concurrent.futures import ProcessPoolExecutor
-from datetime import datetime
 import multiprocessing
 import os
 import posixpath
 import shutil
 import subprocess
-from typing import Optional, List, Tuple
+from concurrent.futures import ProcessPoolExecutor
+from datetime import datetime
+from typing import List, Optional, Tuple
 
```

Comment:
@Leimeroth I assume these changes are just related to committing from an older version and not related to this pull request, correct? Then I would revert those for this pull request.

Reply:
That is related to trying https://docs.astral.sh/ruff/ for linting. Forgot to disable the automatic sorting

Reply:
I adopted the ruff sorting for our continuous integration in https://github.com/pyiron/pyiron_base/pull/1512 so these changes are now already included in the `main` branch.